client/components/plans/plan-price/index.jsx

```diff
@@ -62,13 +65,21 @@ module.exports = React.createClass( {
 			periodLabel = hasDiscount ? this.translate( 'due today when you upgrade' ) : plan.bill_period_label
 		}
 
+		if ( site && site.jetpack ) {
+			return (
+				<JetpackPlanPrice
+					getPrice={ this.getPrice }
+					hasDiscount={ hasDiscount }
+					plan={ plan } />
+			);
+		}
+
 		return (
-			<div className={ hasDiscount ? "plan-price plan-price__discount" : "plan-price" }>
-				<span>{ this.getPrice() }</span>
-				<small className="plan-price__billing-period">
-					{ periodLabel }
-				</small>
-			</div>
+			<WpcomPlanPrice
+				getPrice={ this.getPrice }
+				hasDiscount={ hasDiscount }
+				periodLabel={ periodLabel }
```

Comment:
It looks like this variable can move inside the component rather than being defined in the render.